### PR TITLE
Misc stuff

### PR DESCRIPTION
**Pull request by mbouron-gpsw**

### CODE REVIEW

File: libnodegl/backends/gl/buffer_gl.c [hunk 1]
```diff
@@ -22,7 +22,7 @@
 #include <string.h>
 
 #include "buffer_gl.h"
-#include "gctx_gl.h"
+#include "gpu_ctx_gl.h"
 #include "glcontext.h"
 #include "glincludes.h"
```
Comment (cboesch-gpsw) [inline libnodegl/backends/gl/buffer_gl.c:27]: maintain alphabetical order?

**Review by cboesch-gpsw — APPROVED** [review]
small nit, otherwise LGTM

**Review by jmoguillansky-gpsw — APPROVED** [review]

File: libnodegl/backends/gl/buffer_gl.c (hunk 1 above)
Comment (mbouron-gpsw) [reply libnodegl/backends/gl/buffer_gl.c:27]: There is a lot of inconsistency across the different files I'm modifying. Can this be done globally in a separate commit ?

Comment (cboesch-gpsw) [reply libnodegl/backends/gl/buffer_gl.c:27]: Sure

**Pull request merged**